### PR TITLE
Add job for `R1R*R*R*S*S1S2S3` configuration

### PR DESCRIPTION
**Pull request by ted-miller**
This is to address the topic in https://github.com/ros-industrial/motoman/discussions/538.

### CODE REVIEW

File: motoman_driver/Inform/DX200, FS100, YRC1000/single_arm_with_three_ext_axis/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This particular job configuration is for use with a system that is configured with the `Control Group Extension` function (allow more than two groups in a job). 
+If this function is enabled, the `Group Combination` screen should look like the image shown in `Group_Combination_Settings.png`.
+
+![Group_Combination_Settings](Group_Combination_Settings.png)
+
+If this screen only shows two groups, then the `Control Group Extension` function is not enabled.
```
Comment (gavanderhoorn) [inline motoman_driver/Inform/DX200, FS100, YRC1000/single_arm_with_three_ext_axis/README.md]:
```suggestion
If this screen only shows two groups, then the `Control Group Extension` function is not enabled and the `.JBI` in this directory should not be used.
```

?

**Review by gavanderhoorn — APPROVED** [review]
To me this looks OK.

Perhaps @EricMarcil could also take a look?

**Comment by EricMarcil** [comment]
@gavanderhoorn I looked at it.  It is fine.  Go ahead with the merge.

**Review by EricMarcil — APPROVED** [review]
Look fine

**Pull request merged**